src/site/topics/stacks/stack-adt.rst

```diff
@@ -328,6 +328,11 @@ There has to be a Better Way!
                     Stack<Friend> myFriendStack = new ArrayStack<>();
 
 
+.. warning::
+
+    It is not possible to use primitive types (``int``, ``double``, ``boolean``, etc.) in genetics. The workaround is to
```

Comment:
_generics_, not genetics.

further, it's not a workaround. generics must be objects due to type erasure, primitives are not objects.

Reply:
even the branch has this typo.